app/viewmodels/checkAnswers/transferDetails/assetsMiniJourneys/unquotedShares/UnquotedSharesAmendContinueSummary.scala

```diff
@@ -17,20 +17,20 @@
 package viewmodels.checkAnswers.transferDetails.assetsMiniJourneys.unquotedShares
 
 import controllers.transferDetails.assetsMiniJourneys.AssetsMiniJourneysRoutes
-import models.{CheckMode, Mode, UserAnswers}
+import models.{Mode, UserAnswers}
 import play.api.i18n.Messages
-import queries.assets.UnquotedSharesQuery
+import queries.assets.QuotedSharesQuery
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import uk.gov.hmrc.hmrcfrontend.views.viewmodels.addtoalist.ListItem
+import utils.AppUtils
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
-import utils.AppUtils
-import uk.gov.hmrc.hmrcfrontend.views.viewmodels.addtoalist.ListItem
 
 object UnquotedSharesAmendContinueSummary extends AppUtils {
 
   def row(mode: Mode, userAnswers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] = {
 
-    val answers   = userAnswers.get(UnquotedSharesQuery)
+    val answers   = userAnswers.get(QuotedSharesQuery)
```

Comment:
This page looks correct. I believe the change needs to be here: https://github.com/hmrc/overseas-pension-transfer-frontend/blob/main/app/viewmodels/checkAnswers/transferDetails/assetsMiniJourneys/quotedShares/QuotedSharesAmendContinueSummary.scala#L33